### PR TITLE
Import from ora: provide explicit type mappings via added switch

### LINKED ISSUE #14: Explicitly state types in IMPORT
## Situation

The `IMPORT` command produced by the Oracle Virtual Schema does not enforce types. This means, the ExaLoader picks the types and that in Oracle VS' case leads to numbers being expressed as `VARCHAR`.

## Acceptance Criteria

1. Result types are explicitly set in Oracle VS

### LINKED ISSUE #18: type list
'At the moment we don't supply a type list for the IMPORT if IMPORT_FROM_ORA is used. Only in the JDBC case. Background is the assumption that types don't map well between JDBC and ORA connection. But they'd like to challenge this assumption. Could you create  an experimental branch where you remove the case distinction and try adding the types to the IMPORT statement'

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #14 
Closes #18

### CODE REVIEW

File: doc/changes/changes_2.2.0.md [hunk 1]
```diff
@@ -1,9 +1,24 @@
-# Virtual Schema for Oracle 2.2.0, released 2022-??-??
+# Virtual Schema for Oracle 2.2.0, released 2022-04-06
```
Comment (jakobbraun) [inline doc/changes/changes_2.2.0.md:1]:
```suggestion
# Virtual Schema for Oracle 2.2.0, released 2022-04-11
```

File: pom.xml [hunk 2]
```diff
@@ -210,145 +152,42 @@
                 </configuration>
             </plugin>
```
Comment (jakobbraun) [inline pom.xml:153]: you can remove the compiler plugin now

Comment (jakobbraun) [reply pom.xml:153]: It was added in project-keeper 2.2.0

File: src/main/java/com/exasol/adapter/dialects/oracle/OracleConnectionDefinitionBuilder.java [hunk 3]
```diff
@@ -15,6 +15,7 @@ public class OracleConnectionDefinitionBuilder extends BaseConnectionDefinitionB
     @Override
     public String buildConnectionDefinition(final AdapterProperties properties,
             final ExaConnectionInformation exaConnectionInformation) {
+        // is containing it enough? don't we need to check for true
```
Comment (jakobbraun) [inline src/main/java/com/exasol/adapter/dialects/oracle/OracleConnectionDefinitionBuilder.java:18]: resolve this question

File: src/main/java/com/exasol/adapter/dialects/oracle/OracleQueryRewriter.java [hunk 4]
```diff
@@ -16,13 +21,38 @@
      * @param dialect              Oracle SQl dialect
      * @param remoteMetadataReader reader for metadata from the remote data source
      */
-    public OracleQueryRewriter(final SqlDialect dialect, final RemoteMetadataReader remoteMetadataReader) {
+    Connection connection;
+    AdapterProperties properties;
```
Comment (jakobbraun) [inline src/main/java/com/exasol/adapter/dialects/oracle/OracleQueryRewriter.java:25]: could they be private final?

**Review by jakobbraun — CHANGES_REQUESTED** [review]

File: src/main/java/com/exasol/adapter/dialects/oracle/OracleConnectionDefinitionBuilder.java (hunk 3 above)
Comment (pj-spoelders) [reply src/main/java/com/exasol/adapter/dialects/oracle/OracleConnectionDefinitionBuilder.java:18]: removed the comment

File: src/main/java/com/exasol/adapter/dialects/oracle/OracleQueryRewriter.java (hunk 4 above)
Comment (pj-spoelders) [reply src/main/java/com/exasol/adapter/dialects/oracle/OracleQueryRewriter.java:25]: added private final to both

**Review by jakobbraun — APPROVED** [review]

**Pull request merged**